docs/src/apimanual.md

```diff
@@ -845,6 +842,39 @@ twice the value of the `coefficients` field in the `VectorAffineFunction` for
 the corresponding rows. See [`PositiveSemidefiniteConeTriangle`](@ref) for
 details.
 
+#### Dual for convex problem with quadratic functions
```

Comment:
What about nonconvex problems? I'm motivated by Ipopt which handles nonconvex quadratics. The conditions for a stationary point would be mostly the same, right?

Reply:
Yes, we can remove the `convex` here